profiler/src/Demos/WcfService/Host/WcfServiceHost.csproj

```diff
@@ -53,10 +54,6 @@
       <Project>{b0882e91-443b-4161-ab64-07e188067d3f}</Project>
       <Name>Datadog.Demos.Util</Name>
     </ProjectReference>
-    <ProjectReference Include="..\Library\WcfServiceLibrary.csproj">
```

Comment:
It was not needed any more?
Check that it is still compiling locally

Reply:
👍